src/field_theory/normal_closure.lean

```diff
@@ -0,0 +1,102 @@
+/-
+Copyright (c) 2023 Thomas Browning. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Thomas Browning
+-/
+
+import field_theory.normal
+
+/-!
+# Normal closures
+
+In this file we define the normal closure of an `intermediate_field`.
```

Comment:
Please expand the module docstring so that
- it includes the definition of the normal closure;
- it reminds the reader what is an `intermediate_field` in mathlib;
- it mentions the existing `normal_closure` definition and explains the difference between two definitions.